.gitignore

```diff
@@ -1,5 +1,8 @@
 *.py[cod]
 
+# Kolibri dockerbuild outputs
+kolibribuild/
```

Comment:
This will be very annoying for terminal tab completion :)

Reply:
Is this where we output whl files?

Reply:
Seems like it is. If we do output our whl and tar.gz files in here, I prefer if we keep ignoring it.

Reply:
Not commenting on whether to ignore it or not, it's the directory itself.